content/en/tutorials/artifacts.md

```diff
@@ -189,20 +189,20 @@ To log something as an `Artifact`, we have to first make an `Artifact` object.
 
 Every `Artifact` has a `name` -- that's what the first argument sets.
 
-> **Rule of 👍**: the `name` should be descriptive, but easy to remember and type --
+> **Best practice**: the `name` should be descriptive, but easy to remember and type --
```

Comment:
I think the two hyphens at the end are incorrect. It should probably be a period.